crates/executor/src/error.rs

```diff
@@ -100,6 +101,27 @@ impl From<anyhow::Error> for CallError {
     }
 }
 
+#[derive(Debug)]
+pub struct TransactionExecutorError {
+    pub transaction_index: usize,
+    pub error: BlockifierTransactionExecutorError,
+}
+
+impl TransactionExecutorError {
+    pub fn new(transaction_index: usize, error: BlockifierTransactionExecutorError) -> Self {
+        Self {
+            transaction_index,
+            error,
+        }
+    }
+}
+
+impl std::fmt::Display for TransactionExecutorError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Executor error: {}", self.error)
```

Comment:
Shouldn't it also include `transaction_index`?

Reply:
+1